packages/@guardian/source-foundations/src/colour/palette.stories.mdx

```diff
@@ -0,0 +1,162 @@
+import { Meta } from '@storybook/addon-docs';
+import { palette } from './palette';
+
+<Meta
+	title="Packages/source-foundations/palette"
+	parameters={{ previewTabs: { canvas: { hidden: true } } }}
+/>
+
+# Colour palette
+
+The colour palette is organised by type e.g. `error`, or pillar e.g. `opinion`.
+
+## Example
+
+```tsx
+import { palette } from '@guardian/source-foundations';
+
+const headline = css`
+	color: ${palette.neutral[100]};
+	background: ${palette.brand[400]};
+`;
+
+/*
+.class-name {
+  color: #FFFFFF;
+  background-color: #052962;
+}
+*/
+```
+
+## Colour tokens
+
+<div className="colour_tokens">
+	<div className="colour_tokens__colours__list">
+		<ul>
+			{Object.keys(palette).map((colourCategory, idx) => {
+				const colours = Object.keys(palette[colourCategory]).map(
+					(colour, idx) => {
+						return (
+							<div
+								className="colour_tokens__colours__list__row"
+								key={idx}
+							>
+								<div className="colour_tokens__colours__list__value">
+									<div
+										className="colour_tokens__colours__list__value__swatch"
+										style={{
+											backgroundColor:
+												palette[colourCategory][colour],
+										}}
+									/>
+								</div>
+								<span>
+									<pre>{`palette.${colourCategory}[${colour}]`}</pre>
+								</span>
+							</div>
+						);
+					},
+				);
+				const openFirst = idx === 0;
+				return (
+					<li
+						className="colour_tokens__colours__list__colour_dropdown"
+						key={idx}
+					>
+						<details open={openFirst}>
+							<summary className="colour_tokens__colours__list__category">
+								{colourCategory}
+							</summary>
+							<div>{colours}</div>
+						</details>
+					</li>
+				);
+			})}
+		</ul>
+	</div>
+	<div className="colour_tokens__aside">
```

Comment:
should we make these styles generic rather than specific to colours?

Reply:
Yes, will take a look at that